php/README.md

```diff
@@ -0,0 +1,25 @@
+# Custom Auth with php
+
+The core logic is in `Ionic\CustomAuthenication` class which can be subclassed
```

Comment:
From the look of your class, the authentication details are very engrained. This example seems to be more of an example than a library, which is good, but we just need to change this sentence so people aren't confused when they go to implement it.

For example, many developers won't necessarily need $username and $password. Their authentication scheme may be totally different. There's an example for signing in, but it's a part of the actual class they're told to subclass. 

Ideally we'd have an official PHP SDK to provide this helper, but for now let's make the example more of an example than a library.